app/src/main/java/org/schabi/newpipe/fragments/detail/VideoDetailFragment.java

```diff
@@ -544,10 +545,14 @@ public boolean onLongClick(final View v) {
 
         switch (v.getId()) {
             case R.id.detail_controls_background:
-                openBackgroundPlayer(true);
+                if (PlayerHolder.getInstance().isPlayerOpen()) {
+                    openBackgroundPlayer(true);
+                }
```

Comment:
I'd go with removing it as also suggested in [this comment of yours](https://github.com/TeamNewPipe/NewPipe/pull/6872#issuecomment-899794880) (for future reference).

Reply:
note for myself: the openBackgroundPlayer(boolean append) method works within the current videoDetailfragment and appends the current video if true.

Reply:
> Just to be sure, please make sure that the `if (player is open)` **before enqueueing** is removed everywhere,

I am assuming that this refers to those `if (player is open)` **before enqueueing** that are called by long pressing a button.
Which would imply changes to the LocalPlaylistFragment, PlaylistFragment and VideoDetailFragment (handleResult, onLongClick).

> and that the play next action is available in every long press menu of each fragment.

assuming that this refers to the FeedFragment, StatisticsPlaylistFragment, LocalPlaylistFragment, PlaylistFragment and their Dialogs; mostly called showStreamDialog and showStreamItemDialog

I will be applying the requested changes to the listed Fragments that I have discovered. If i have missed some please just notify me @Stypox.

P.S.; I intend on opening a new issue (or directly a PR) which merges and unifies all these into one extendable base (Fragment?, List?). Not quiet sure of the implementation, just pretty sure that it needs refactoring, since its a huge pain to work with. Indicated by how many times I've missed that code needs to be duplicated and added to another fragment as well.